src/cookieSync.js

```diff
@@ -28,15 +30,26 @@ function doBidderSync(type, url, bidder, done) {
     console.log(`Invoking image pixel user sync for bidder: "${bidder}"`);
     triggerPixel(url, done);
   } else if (type == 'iframe') {
-    console.log(`Skipping iframe pixel user sync for bidder: "${bidder}". This isn't implemented yet.`);
-    // TODO test iframe solution
-    done();
+    console.log(`Invoking iframe pixel user sync for bidder: "${bidder}"`);
+    triggerIframeLoad(url, bidder, done);
   } else {
     console.log(`User sync type "${type}" not supported for bidder: "${bidder}"`);
     done();
   }
 }
 
+function triggerIframeLoad(url, bidder, done) {
+  let iframe = domHelper.getEmptyIframe(0, 0);
+  iframe.id = `sync_${bidder}_${Date.now()}`;
```

Comment:
shouldn't these 2 statements be after the the `if` condition check for url

Reply:
yeah good point